pkg/docker/config/config_test.go

```diff
@@ -280,12 +309,19 @@ func TestGetAuth(t *testing.T) {
 				if tc.sys != nil {
 					sys = tc.sys
 				}
-				auth, err := getCredentialsWithHomeDir(sys, tc.hostname, tmpHomeDir)
+
+				var ref reference.Named
+				if tc.ref != "" {
+					ref, err = reference.ParseNamed(tc.ref)
+					assert.Nil(t, err)
+				}
+
+				auth, err := getCredentialsWithHomeDir(sys, ref, tc.hostname, tmpHomeDir)
 				assert.Equal(t, tc.expectedError, err)
 				assert.Equal(t, tc.expected, auth)
 
 				// Test for the previous APIs.
-				username, password, err := getAuthenticationWithHomeDir(sys, tc.hostname, tmpHomeDir)
+				username, password, err := getAuthenticationWithHomeDir(sys, ref, tc.hostname, tmpHomeDir)
```

Comment:
This is not really representative of anything. Test with `nil` and do… something, maybe just ignore the return value if the test case doesn’t record the expected data, or another field to the test case.

Reply:
Removed the added parameter from this function so we now can preserve the original test case.